### PR TITLE
Update readMe, user guide and developer guide

### PR DESCRIPTION
**Pull request by yiwen101**
#27,#38, #40, #28

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,14 +1,21 @@
...
+TutorMate is a desktop app targeted to tuition teachers for managing contacts of tuition students. 
+
+It is optimized for use via a Command Line Interface (CLI) while still having the benefits of a Graphical User Interface (GUI). If you can type fast, TutorMate can get your administrative tasks done faster than traditional GUI apps.
+
+This project is based on the [AddressBook-Level3 project](https://se-education.org).
+
+## Quick Start
+1. Ensure you have Java `11` or above installed in your Computer.
+2. Go to this installation package from this [release page](todo). 
+3. Download the .dmg file for Mac OS, .exe file for Windows, and .jar file for Linux.
```
Comment (richiehx) [inline README.md:14]: Should the files all just be jar files for the different Operating Systems?

File: README.md [hunk 2]
```diff
@@ -1,14 +1,21 @@
...
+
+[![CI Status](https://github.com/AY2324S1-CS2103T-T11-3/tp/actions/workflows/gradle.yml/badge.svg)](https://github.com/AY2324S1-CS2103T-T11-3/tp/actions)
+
+TutorMate is a desktop app targeted to tuition teachers for managing contacts of tuition students. 
+
+It is optimized for use via a Command Line Interface (CLI) while still having the benefits of a Graphical User Interface (GUI). If you can type fast, TutorMate can get your administrative tasks done faster than traditional GUI apps.
+
+This project is based on the [AddressBook-Level3 project](https://se-education.org).
+
+## Quick Start
```
Comment (richiehx) [inline README.md:11]: Should the Quick Start be in the User Guide instead of the ReadMe?

File: docs/DeveloperGuide.md [hunk 3]
```diff
@@ -314,6 +314,24 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
 (For all use cases below, the **System** is the `AddressBook` and the **Actor** is the `user`, unless specified otherwise)
 
+**Use case: Install the app**
+MSS 
+1. User execute the installation package
+2. The app ask for continue or cancel with a pop up window
+3. User click continue
+4. The app install the app
+5. The app show a pop up window to inform user the installation is successful
```
Comment (richiehx) [inline docs/DeveloperGuide.md]: Perhaps the Use Case can be reformatted to remove UI details. I feel that point 4 is not very clear.

File: docs/DeveloperGuide.md [hunk 4]
```diff
@@ -314,6 +314,24 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
...
+Use Case ends
+
+Extensions
+* 3a. User click cancel
+  * 3a1. The app close the installation package
```
Comment (richiehx) [inline docs/DeveloperGuide.md]: Same as above, perhaps the extension could be just: User cancels the Installation

**Review by richiehx — COMMENTED** [review]
LGTM! However, there are some minor questions and comments regarding the content. I feel that the use cases could be refactored to remove the UI details.

File: README.md (hunk 2 above)
Comment (yiwen101) [reply README.md:11]: I believe ReadMe should be a subSet of user Guide, is not it?

File: docs/DeveloperGuide.md (hunk 4 above)
Comment (yiwen101) [reply docs/DeveloperGuide.md]: sure, will fix that

**Review by pzl111 — APPROVED** [review]
LGTM

**Pull request merged**